projects/igniteui-angular/src/lib/directives/dragdrop/README.md

```diff
@@ -0,0 +1,117 @@
+# igxDrag
+**igxDrag** is a directive that enables dragging of elements around the page.
+
+## Usage
+```html
+<div igxDrag [hideBaseOnDrag]="true" [animateOnRelease]="true" *ngFor="let elem of draggableElems" >
+    <span [style.margin]="'auto'">{{elem.label}}</span>
+</div>
+```
+
+## Getting Started
+
+### Introduction
+
+When an element inside your Angular application needs to be dragged from one place to another the `igxDrag` directive can be used to achieve this behavior. In combination with the `igxDrop` directive the placing of the dragged element can be done as well to have fully interactive application.
+
+### Basic configuration
+
+The `igxDrag` directive can be applied on any DOM element by just adding it to its template.
+
+```html
+<div igxDrag>Drag me</div>
+```
+
+### DOM Behavior
+
+By default when an element with `igxDrag` directive applied needs to be dragged the user needs to swipe at least 5px in any direction in order for the actual dragging to occur. Otherwise the interaction is considered as a click and the `dragClicked` event is emitted.
+
+When dragging occurs a drag ghost element is spawned that is positioned so it is always under the mouse cursor or when on touch where it is being touched. The original element is still present, but it can be hidden automatically when dragging starts with the `hideBaseOnDrag` input. 
```

Comment:
Suggestion:
When dragging occurs a drag ghost element is spawned and moves along with the mouse cursor or touch interaction.